ci/github-script/README.md

```diff
@@ -0,0 +1,13 @@
+# GitHub specific CI scripts
+
+This folder contains `actions/github-script`-based JavaScript code.
```

Comment:
Let's link to actions/github-script so we can quickly jump there when reading this